main.go

```diff
@@ -65,6 +68,12 @@ func runDocker2ACI(arg string, flagNoSquash bool, flagImage string, flagDebug bo
 		return fmt.Errorf("conversion error: %v", err)
 	}
 
+	if squash {
```

Comment:
I can add it. It looks a bit ugly because there will be a lot of repeated volumes:

```
Converted volumes in "postgres-26bdc4facea7aad8cfdf3da846e3767bfa027fc0b3c621501f8d7a2413fd24e2-latest-linux-amd64.aci":
        name: "volume-/var/lib/postgresql/data", path: "/var/lib/postgresql/data", readOnly: false

Converted volumes in "postgres-3508d36eab9991998d587938711fd19286205eb1a7e994fc4df804ab08e14013-latest-linux-amd64.aci":
        name: "volume-/var/lib/postgresql/data", path: "/var/lib/postgresql/data", readOnly: false

Converted volumes in "postgres-86cff6f79a0869259cd911587f61282d9c7a5afc87ca425290cfc781cfaa83bd-latest-linux-amd64.aci":
        name: "volume-/var/lib/postgresql/data", path: "/var/lib/postgresql/data", readOnly: false

Converted volumes in "postgres-9c5489b55573bdc2e214f35ed102543845c0de6ca5df10eb5c037ee93a126002-latest-linux-amd64.aci":
        name: "volume-/var/lib/postgresql/data", path: "/var/lib/postgresql/data", readOnly: false

Converted volumes in "postgres-dcce0da1c31105323a840ded6e79f39b7338907f5c2477109b68692e7acb75be-latest-linux-amd64.aci":
        name: "volume-/var/lib/postgresql/data", path: "/var/lib/postgresql/data", readOnly: false
```